### ISSUE TITLE
 Debug keystore not configured in the Editor Settings nor in the preset.

### ISSUE THREAD

**Issue by AlexRazor1337** [bend-n/godot-actions#2]
Hi! I'm using the workflow like in "Full example" in readme. And on Android export I have this error.
```
ERROR: Cannot export project with preset "Android" due to configuration errors:
Debug keystore not configured in the Editor Settings nor in the preset.
```
Any idea why? I've looked at your godot-template repo and I have everything like there.

**Comment by AlexRazor1337** [comment]
So looked through the logs more carefully and found:
```
Downloading Android SDKs
  keytool error: java.lang.Exception: Key pair not generated, alias <androiddebugkey> already exists
 ```
 
So silly me, forgot to remove keystore from old tries from the repo!

**Comment by bend-n** [comment]
Im unsure what happened, and how I can fix/document this behaviour. Could you elaborate?

**Comment by AlexRazor1337** [comment]
I've had `debug.keystore` file in my repo, I've added it at some point while trying to make different Godot GitHub actions work. Then I've switched to this repo actions and had that error on Android Debug build because it was trying to generate new keystore while previous still existed.

To fix/document, I propose explicitly throw an error if there is a `debug.keystore` file already or use this file instead of generating new one.
